container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/fluentbit_logs_collector_daemonset.go

```diff
@@ -0,0 +1,461 @@
+package fluentbit
```

Comment:
The bulk of the important stuff for fluent bit configuration happens here -namespace is created , then config map to hold the configuration, then daemon set.